src/main.ts

```diff
@@ -24,7 +25,21 @@ export async function run(): Promise<void> {
       }
       identityUrl = baseUrl + "/identity";
       apiUrl = baseUrl + "/api";
+    } else {
+      if (!identityUrl && !apiUrl) {
```

Comment:
@kimdre thank you for addressing our concerns, you rock! 😄 I just changed it a teeny bit so that it handles the scenario when identityUrl OR apiUrl are set and we don't want it overwritten. 
  
//If IdentityUrl or ApiUrl does not have a URL defined, we need to create one based off the cloudRegion
  if (!identityUrl || !apiUrl) {
    let cloudBaseUrl: string;

    switch (cloudRegion) {
        case "us":
            cloudBaseUrl = "bitwarden.com";
            break;
        case "eu":
            cloudBaseUrl = "bitwarden.eu";
            break;
        default:
            throw new TypeError("Input provided for cloud_region is not in the expected format");
    }

    //If identityUrl doesn't have a default, set it using the cloudBaseUrl
    if (!identityUrl) {
        identityUrl = `https://identity.${cloudBaseUrl}`;
    }

    //If apiUrl doesn't have a default, set it using the cloudBaseUrl
    if (!apiUrl) {
        apiUrl = `https://api.${cloudBaseUrl}`;
    }
}

Reply:
@kimdre if you agree with my suggestion, please make that final change, and I will test everything on my end before proceeding with the next steps! 🥳

Reply:
@mzieniukbw since you are reviewing this as well, how do you feel about my suggestion? Should we allow the user to define only identityUrl/apiUrl and not define both?

Reply:
I prefer we error early on override cases, since it would be confusing to the user on what's the expectancy in this scenario. Simpler is better.
I propose to add an additional validation to:
- error when `cloud_region` and any of the self-hosted inputs are set at the same time.
- error when `base_url` and any of the `identity_url` or `api_url` inputs are set at the same time.
- error when `identity_url` or `api_url` input is set, but not the other one - i.e. when `identity_url` is set, but not `api_url`; or when `api_url` is set, but not `identity_url`